controllers/TopicsControllers.js

```diff
@@ -0,0 +1,13 @@
+const {fetchTopics} = require('../models/topicsModel')
+
+const getTopics = (req, res, next) => {
+    fetchTopics()
+        .then((topics) => {
+            res.status(200).send(topics.rows);
```

Comment:
all data from your controllers should be sent back on a meaningful key ie. `.send({topics: topics.rows})`